plugins/lookup/aws_secret.py

```diff
@@ -70,7 +95,7 @@
 from ansible.plugins import AnsiblePlugin
 from ansible.plugins.lookup import LookupBase
 from ansible.module_utils._text import to_native
-
+from ansible_collections.amazon.aws.plugins.module_utils.core import is_boto3_error_code
```

Comment:
```suggestion
from ansible_collections.amazon.aws.plugins.module_utils.core import is_boto3_error_code

```
Removal of the double blank line before starting to define functions is triggering a linter warning